webapp/app/.server/service/clients.ts

```diff
@@ -4,4 +4,3 @@ import { BlockExplorerClient, RpcClient } from "validate-cli";
 export const l1Explorer = BlockExplorerClient.forL1(env.ETHERSCAN_API_KEY, env.ETH_NETWORK);
 export const l1Rpc = new RpcClient(env.L1_RPC_URL);
 export const l2Explorer = BlockExplorerClient.forL2(env.ETH_NETWORK);
-export const l1RpcProposals = new RpcClient(env.L1_RPC_URL_FOR_UPGRADES);
```

Comment:
Now there is only l1Rpc. This change involves several changes in other files, that where using one or the other.